uk.co.compucorp.civicrm.hrcore/CRM/HRCore/Menu/MainMenuConfig.php

```diff
@@ -0,0 +1,179 @@
+<?php
+
+class CRM_HRCore_Menu_MainMenuConfig implements CRM_HRCore_Menu_Config {
+
+  /**
+   * Returns menu Items for Main navigation menu Items.
+   *
+   * @return array
+   */
+  public function getItems() {
+    return [
+      'Home' => 'civicrm/tasksassignments/dashboard#/tasks',
+      'Search' =>
+        [
+          'icon' => 'crm-i fa-search',
+          'children' =>
+            [
+              'Find Contacts' => 'civicrm/contact/search?reset=1',
+              'Advanced Search' => 'civicrm/contact/search/advanced?reset=1',
+            ],
+        ],
+      'Staff' =>
+        [
+          'icon' => 'crm-i fa-users',
+          'children' =>
+            [
+              'New Individual' =>
+                [
+                  'url' => 'civicrm/contact/add?reset=1&ct=Individual',
+                  'permission' => 'add contacts',
+                ],
+              'New Organization' =>
+                [
```

Comment:
I think we can and we should reduce the number of lines in these files, also, maybe, add some empty lines to improve readability. For example, by simply not putting the opening `[` on a new line will reduce the number of lines in these array by A LOT. For example, check this comparison between a small fraction of the file: https://gist.github.com/davialexandre/539444362f785ed3127df08707de4c51

As you can see, the compact version has 9 fewer lines than the original version, even though it includes empty lines between some items to improve the readability.

Could you please ask the other devs what they think about this?

Reply:
I agree we can improve readability by adding empty lines in between and not putting the opening `[` on a new line. I'll do that.